solr/solrj/src/java/org/apache/solr/client/solrj/io/stream/CloudSolrStream.java

```diff
@@ -334,11 +334,6 @@ private StreamComparator parseComp(String sort, String fl) throws IOException {
   public static Slice[] getSlices(String collectionName, ZkStateReader zkStateReader, boolean checkAlias) throws IOException {
     ClusterState clusterState = zkStateReader.getClusterState();
 
-    Map<String, DocCollection> collectionsMap = clusterState.getCollectionsMap();
-
-    //TODO we should probably split collection by comma to query more than one
-    //  which is something already supported in other parts of Solr
-
     // check for alias or collection
```

Comment:
Should we cache the value of `zkStateReader.getAliases` below to avoid volatile reads?

Reply:
Moved the call to getAliases out of the for loop